server.go

```diff
@@ -412,6 +412,7 @@ type Server struct {
 	concurrencyCh    chan struct{}
 	perIPConnCounter perIPConnCounter
 	serverName       atomic.Value
+	FormValueFunc    FormValueFunc
```

Comment:
This field needs some documentation since it's exposed.

Reply:
thanks and done